src/components/UI/Checkbox.js

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const CheckboxWrapper = styled.div``;
+
+const CheckboxLabel = styled.span`
+  margin-left: 5px;
+`;
+
+/* eslint-disable-next-line no-empty-function */
+const Checkbox = ({ checked, label, onChange = () => {} }) => {
+  return (
+    <CheckboxWrapper onClick={() => onChange(!checked)}>
+      <i
+        className={`fa fa-${checked ? 'check-box' : 'check-box-outline-blank'}`}
+      />
+      <CheckboxLabel className='checkbox-label'>{label}</CheckboxLabel>
+    </CheckboxWrapper>
+  );
```

Comment:
This approach is not accessible, nor compatible with the native checkbox.
Custom appearances for checkboxes can be created with just CSS.

We can work together on that in a separate PR, as we should move on with this one.